docs/how-to/uc-boot-parameters.rst

```diff
@@ -0,0 +1,89 @@
+How to configure boot parameters on Ubuntu Core 
+================================================
+
+.. note::
+
+    This guide assumes access to features available exclusively to `dedicated Snap Store`_ users.
+
+Ubuntu Core doesn't have the :code:`/etc/default/grub` file mapped for the kernel parameters, nor the `update-grub`_ utility.
+On Ubuntu Core, the kernel parameters can be modified statically or dynamically.
+Static configurations are added during the image build.
+On the other hand, dynamic configurations are set `using snap options`_ on a running system.
+This guide focuses on the dynamic configuration method, suitable for iteratively tuning the operating system.
+
+.. hint::
+    
+    Refer to :doc:`../how-to/uc-image-creation` if you want to create a real-time Ubuntu Core image.
+
+Open a terminal and access your real-time Ubuntu Core instance.
+The instance should be on a physical device rather than a virtual machine.
+While a virtual machine can be used to follow the steps in this guide, it will not provide the benefits of a real-time kernel.
```

Comment:
Hi @farshidtz , not sure if I'm misunderstanding the intent of this line because the VM won't be 'providing' the benefits but won't be able to 'inherit' the benefits.

```suggestion
While a virtual machine can be used to follow the steps in this guide, it will not benefit from the features of a real-time kernel.
```

Reply:
I think the existing explanation is ok. Running real-time workload in a VM involves the host OS, hypervisor, and the VM environment. I'll leave this for @locnnil to decide, since this text was written by him in #39.

Reply:
I like @AnneCYH suggestion! That's what I meant to say, it could be used  "characteristics" as well. Applied on https://github.com/canonical/real-time-ubuntu-docs/pull/45/commits/1d8d02797da0424c35b335a1fd758a6a8f5d4a20. Thanks!